examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+subjects_dir = data_path + '/subjects'
+
+# read precomputed data
+compute_lcmv_example_data_surf(data_path, fname='lcmv-surf')
+compute_lcmv_example_data_vol(data_path, fname='lcmv-vol')
+
+src = mne.read_forward_solution(fname_fwd)['src']
+stc = mne.read_source_estimate('lcmv-vol-vl.stc')
+stc_rh = mne.read_source_estimate('lcmv-surf-rh.stc')
+
+###############################################################################
+# Morph VolSourceEstimate
+
+# initialize morpher for VolSourceEstimate (src=src)
+morpher = SourceMorph('sample',
+                      'fsaverage',
+                      src=src,
+                      subjects_dir=subjects_dir,
+                      grid_spacing=(5., 5., 5.))
+
+# save morph
+morpher.save('vol')
+
+# load morph by initializing with file name
+morpher_loaded = SourceMorph(None, None, fname='vol-morph.h5')
+
+# morph data
+stc_morphed = morpher_loaded(stc)
+
+###############################################################################
+# Morph SourceEstimate
+
+fs_vertices = [np.arange(10242)] * 2
+
+# initialize morpher for SourceEstimate (src='surf')
+morpher_rh_surf = SourceMorph('sample', 'fsaverage',
+                              data_from=stc_rh.vertices,
+                              data_to=fs_vertices,
+                              subjects_dir=subjects_dir,
+                              src='surf')
+
+# save morph
+morpher_rh_surf.save('surf_rh')
+
+# load morph by initializing with file name
+morpher_rh_surf_loaded = SourceMorph(None, None, None,
+                                     fname='surf_rh-morph.h5')
+
+# morph data
+stc_rh_morphed = morpher_rh_surf_loaded(stc_rh)
+
+###############################################################################
+# Plot results
+
+# plot morphed surface
+# vertno_max, time_max = stc_rh_morphed.get_peak(hemi='rh')
+
+# surfer_kwargs = dict(
+#     hemi='rh', subjects_dir=subjects_dir,
+#     clim=dict(kind='value', lims=[8, 12, 15]), views='lateral',
+#     initial_time=time_max, time_unit='s', size=(800, 800), smoothing_steps=5)
+
+# brain = stc_rh_morphed.plot(**surfer_kwargs)
+# brain.add_text(0.1, 0.9, 'Morphed to fsaverage', 'title', font_size=20)
+
+# plot morphed volumes
+
+# create mri-resolution volume of results
+img = morpher_loaded.as_volume(stc_morphed, mri_resolution=True)
+
+figure, axes = plt.subplots()
+figure.subplots_adjust(top=0.8, left=0.1, right=0.9, hspace=0.5)
+figure.patch.set_facecolor('black')
+
+display = plot_anat(nib.load(t1_fsaverage),
+                    display_mode='ortho',
+                    cut_coords=[0., 0., 0.],
+                    draw_cross=False,
+                    axes=axes,
+                    figure=figure,
+                    annotate=False)
+
+display.add_overlay(img, alpha=0.75)
+display.annotate(size=8)
+axes.set_title('morphed subject results to fsaverage', color='white',
+               fontsize=12)
+plt.text(plt.xlim()[1], plt.ylim()[0], 't = 0.087s', color='white')
+plt.show()
```

Comment:
example should be ideally less than 100 lines. Otherwise, you lose the user ... 

and the code reviewer :)

Reply:
err... - I can try :D that's the best I can offer